integration_test/e2e-ui-tests/src/tests/SettlementsPage.test.ts

```diff
@@ -226,6 +226,14 @@ test.meta({
     expectedAccountState,
     'All participant settlement account balances and NDCs should have been set correctly'
   );
+
+  // Open the detail modal to check if DFSP name is rendered for the settlement
+  await t.click(Selector(settlementRowAfter.id))
+  const settlementDetailRows = await SettlementDetailModal.getWindowsRows()
+  const dfspNames = await Promise.all(settlementDetailRows.map((r: WindowRow) => r.dfsp.innerText));
+  await t.expect(dfspNames.includes(participants[0].name)).ok()
+  await t.expect(dfspNames.includes(participants[1].name)).ok()
```

Comment:
@kleyow  why there are 2 participants? Are 2 participants getting added as part of these tests?